app/v1/controllers/ViewReturnController.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers
+
+import cats.data.EitherT
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import play.mvc.Http.MimeTypes
+import utils.{EndpointLogContext, Logging}
+import v1.controllers.requestParsers.ViewReturnRequestParser
+import v1.models.errors._
+import v1.models.request.viewReturn.ViewRawData
+import v1.services.{EnrolmentsAuthService, ViewReturnService}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ViewReturnController @Inject()(val authService: EnrolmentsAuthService,
+                                    requestParser: ViewReturnRequestParser,
+                                    service: ViewReturnService,
+                                    cc: ControllerComponents)(implicit ec: ExecutionContext)
+  extends AuthorisedController(cc) with BaseController with Logging {
+
+  implicit val endpointLogContext: EndpointLogContext =
+    EndpointLogContext(
+      controllerName = "VatReturnsResource",
+      endpointName = "retrieveVatReturns"
+    )
+
+  def viewReturn(vrn: String, periodKey: String): Action[AnyContent] =
+    authorisedAction(vrn).async{ implicit request =>
+      logger.debug(s"[VatReturnsResource] [retrieveVatReturns] Retrieve VAT returns for VRN : $vrn")
```

Comment:
Should this log be [Class] [Method]?

Reply:
this is replicating the logging in legacy VAT

Reply:
The exact log doesn't have to be replicated, just the message is important probably

Reply:
I'll keep the message and update the [Class][Method] to be accurate to the new code.